ddl/table.go

```diff
@@ -203,7 +203,8 @@ func onTruncateTable(t *meta.Meta, job *model.Job) (ver int64, _ error) {
 		job.State = model.JobStateCancelled
 		return ver, errors.Trace(err)
 	}
-	tblInfo, err := getTableInfo(t, job, schemaID)
```

Comment:
Why change here?

Reply:
before rebase ...`go tool vet` will say this shadow `err`...but after rebase complain is disappeared, orz....

reverted..